relay/sources/rudder-server-relay-postinst

```diff
@@ -27,13 +27,11 @@ if ! getent passwd rudder >/dev/null; then
   echo " Done"
 fi
 
-# Reload systemd before managine the services
-if [ -z "${FIRST_INSTALL}" ]; then
-  echo -n "INFO: Setting rudder-relayd as a boot service..."
-  systemctl daemon-reload
-  systemctl enable rudder-relayd
-  echo " Done"
-fi
+# Always reload systemd (in case we changed the unit file)
+echo -n "INFO: Setting rudder-relayd as a boot service..."
+systemctl daemon-reload
+systemctl enable rudder-relayd
```

Comment:
I think we only want to enable it at first install, people may wish to disable it temporarily and upgrade shouldn't change service status